### PR TITLE
add LoginWithKeycard to mobile pkg

### PR DESCRIPTION
**Pull request by gravityblast**
This function is already present in `lib/library.go`, and now in the `mobile` pkg as well.

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

**Comment by mandrigin** [comment]
I will build a release out of this branch to be able to re-launch the CI on status-react.

**Comment by mandrigin** [comment]
merging because the change is trivial and was just forgotten

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.21.0g-beta.1
+0.21.0g-beta.2
```
Comment (adambabik) [inline VERSION:1]: Any idea what does a letter `g` do in this version?

Comment (gravityblast) [reply VERSION:1]: @adambabik I'm not sure tbh, I just bumped the version

Comment (gravityblast) [reply VERSION:1]: maybe @mandrigin knows?

**Pull request merged**